### PR TITLE
Client-to-client inquiries

### PR DESCRIPTION
**Pull request by kythyria**
This proposes a replacement for the parts of CTCP that are not messages intended for human consumption and therefore are somewhat strange to place in a PRIVMSG.

### CODE REVIEW

**Comment by kaniini** [comment]
I think that this should be merged into 3.3 intents.  Thoughts?

**Comment by clokep** [comment]
I think this looks pretty reasonable and would likely fit some use cases I have! Are there any thoughts on whether an `INQUIRY` can result in multiple `IQRESULT` messages?

File: extensions/inquiries [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+command. The arguments are whatever response is appropriate for the command.
+ 
+## Sending to channels
+
+Clients MAY send INQUIRE messages with channels as the target, subject to normal
+access control for the channel in question. As with PRIVMSG, the server forwards
+the message to all other users in the channel. A client receiving such a message
+SHOULD respond in the same manner as if the inquiry had been sent directly to it.
+
+Clients MUST NOT send IQRESULT messages to channels. Servers MUST respond to any
```
Comment (SaberUK) [inline extensions/inquiries]: Would it not be better to replace `<Target>` with `<Nick>` instead of explicitly rejecting channels?

**Comment by jwheare** [comment]
Closing via #270

**Pull request closed without merging**